validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -246,7 +301,8 @@ def __init__(self,
                     identity=identity,
                     secured=secured,
                     server_public_key=server_public_key,
-                    server_private_key=server_private_key)
+                    server_private_key=server_private_key,
+                    heartbeat=heartbeat)
```

Comment:
But heartbeat is True for the network instance set in core.py and that True is passed in on every Connection. It should probably just not be passed in to Connection.